registry.json

```diff
@@ -26,6 +26,33 @@
             "downloadURL": "https://github.com/SparedRay/superpowers-game-SparedRay-easystarjs-plugin/releases/download/v0.1.0/superpowers-easystarjs-api.zip",
             "releaseNotesURL": "https://github.com/SparedRay/superpowers-game-SparedRay-easystarjs-plugin/releases"
           }
+        },
+        "florentpoujol": {
+          "fText": {
+            "version": "0.4.0",
+            "downloadURL": "https://github.com/florentpoujol/superpowers-game-ftext-plugin/archive/v0.4.0.zip",
+            "releaseNotesURL": "https://github.com/florentpoujol/superpowers-game-ftext-plugin/releases"
+          },
+          "fMouseInput": {
+            "version": "0.4.0",
+            "downloadURL": "https://github.com/florentpoujol/superpowers-game-fmouseinput-plugin/archive/v0.4.0.zip",
+            "releaseNotesURL": "https://github.com/florentpoujol/superpowers-game-fmouseinput-plugin/releases"
+          },
+          "fLang": {
+            "version": "0.4.1",
+            "downloadURL": "https://github.com/florentpoujol/superpowers-game-flang-plugin/archive/v0.4.1.zip",
+            "releaseNotesURL": "https://github.com/florentpoujol/superpowers-game-flang-plugin/releases"
+          },
+          "DOM": {
+            "version": "0.4.0",
+            "downloadURL": "https://github.com/florentpoujol/superpowers-game-dom-plugin/archive/v0.4.0.zip",
+            "releaseNotesURL": "https://github.com/florentpoujol/superpowers-game-dom-plugin/releases"
+          },
+          "THREE.js": {
```

Comment:
The link makes you download a zip which contain a folder vith dots (ie : superpowers-game-dom-plugin-0.4.0 )

So yeah, if folders with dots in their name are ignored, it won't work.
Unless (maybe) if I name my tags like v0-4-0 or add a separate download file like you do.
I'm gonna go with this last one I guess.